index.html

```diff
@@ -78,20 +88,20 @@ <h1>My Recent Works</h1>
                         <div class="post">
                             <h2>Multi-Post Stories</h2>
                             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                        </div>
-                        <ul class="tags">
-                            <li class="card-tag-crystal">css</li>
-                            <li class="card-tag-crystal">html</li>
-                            <li class="card-tag-crystal">bootstrap</li>
-                            <li class="card-tag-crystal">ruby</li>
-                        </ul>
-                        <div class="mpfooter">
-                            <a href="#">See Project</a>
+                            <ul class="tags">
+                                <li class="card-tag-crystal">css</li>
+                                <li class="card-tag-crystal">html</li>
+                                <li class="card-tag-crystal">bootstrap</li>
+                                <li class="card-tag-crystal">ruby</li>
+                            </ul>
+                            <div class="mpfooter">
+                                <a href="#">See Project</a>
+                            </div>
                         </div>
                     </div>
```

Comment:
- [ ] The featured project image should take 60% of the page's width and is aligned to the left, the description should take the remaining 40% and is aligned to the right. Starting from screen size 950px, the project section of your portfolio is cut off at the two edges and a horizontal navigation bar automatically shows up. This implies that your elements have a combined width that extends beyond the viewport. This does not make the page look good. Please let's fix it.
- [ ] The contents of the featured project description should be aligned to the left.
- [ ] There's a `See Project` button that should take you to the Live Demo of the featured project or to the GitHub repository.
- [ ] When the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears to the bottom of the image and takes the whole image width. The mentioned button should take you to the Live Demo of the project or to the GitHub repository.
- [ ] The list of technologies in the `Language, Frameworks and SKills` section of the "About Me" section of your portfolio are wobbly and not well arranged. In the template, they are aligned to the left in a row under each heading. Please let's fix it. 
- [ ] The bold text in beside the contact form should start on the same line with the form just as it is in the template.

Reply:
- [ ] As the previous reviewer mentioned. For each displayed project, the description of each of these projects is on top of their image with a dark opacity with white color text. when the project is hovered on with the mouse the description and dark opacity disappear and a See Project button appears. Kindly fix the issue, you can achieve this using CSS by changing the display's value on :hover of these elements.
![image](https://user-images.githubusercontent.com/43698511/130331283-3896dcdf-dc57-49f7-9277-0dd28535a7aa.png)